src/mesh/mesh_refinement.cpp

```diff
@@ -50,10 +50,11 @@ MeshRefinement::MeshRefinement(std::weak_ptr<MeshBlock> pmb, ParameterInput *pin
   // Create coarse mesh object for parent grid
   coarse_coords = Coordinates_t(pmb.lock()->coords, 2);
 
-  if (NGHOST % 2) {
+  if (Globals::nghost % 2) {
```

Comment:
For clarity, I suggest:
```suggestion
  if ((Globals::nghost % 2) != 0) {
```